pkg/test/case_integration_test.go

```diff
@@ -21,14 +21,14 @@ func TestMultiClusterCase(t *testing.T) {
 		t.Error(err)
 		return
 	}
-	defer testenv.Environment.Stop()
+	t.Cleanup(testenv.Environment.Stop)
```

Comment:
```suggestion
	t.Cleanup(func() {
		if err := testenv.Environment.Stop(); err != nil {
			t.Error(err)
		}
	})
```

This doesn't compile... param is `func() {` and `Stop` is of type `func() error {` this was likely missed due to this file being `go:build integration`

Reply:
Hmmm, shouldn't we at least try to compile in CI ?